### PR TITLE
V1 design | MVP splash page

### PR DESCRIPTION
**Pull request by Da-Colon**
## Description
First in a series of PRs to replace the current MVP components with Chakra-UI components, using similar designs to MVP. The goal is to change the UI as little as possible but get app back into a testable state for the USUL modules, as well as get to a place where we can focus on the updates to the new DAO screens

## Screenshots (if applicable)
image

### CODE REVIEW

**Review by Da-Colon — COMMENTED** [review]
No crazy new changes just reskinning

File: src/routes/FractalRoutes.tsx [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
       <Route
         index
         element={<Home />}
       />
-      <Route
+      {/* <Route
         path="daos/*"
         element={<DAOs />}
-      />
+      /> */}
```
Comment (herbig) [inline src/routes/FractalRoutes.tsx:14]: why is this commented out?

File: src/pages/Home/index.tsx [hunk 2]
```diff
@@ -1,41 +1,52 @@
...
-            <SecondaryButton
-              id="home:link-find"
-              label={t('buttonFind')}
-              isLarge
-              className="w-full"
-            />
+          <Link to="/daos">
+            <Button
+              variant="secondary"
+              data-testid="home-linkFind"
```
Comment (herbig) [inline src/pages/Home/index.tsx:40]: all these test ids are going to make setting up the text infrastructure incredibly easy 🔥

**Review by herbig — APPROVED** [review]
LGTM

**Pull request merged**